apps.yaml

```diff
@@ -292,3 +292,17 @@ qemu-web-desktop:
         - utilities
         - technology-simstack
         - data-analysis
+nmrium:
+    metadata:
+        title: NMRium
+        description: |
+            NMRium allows to display 1D and 2D NMR spectra directly from the browser. NMR spectra should be stored in the JCAMP-DX format.
```

Comment:
I would replace by "NMRium is a tool for visualizing 1D and 2D NMR spectra that are stored in JCAMP-DX format directly in your web browser.".